courseware/models.py

```diff
@@ -20,3 +21,16 @@ class CoursewareUser(TimestampedModel):
 
     class Meta:
         unique_together = ("user", "platform")
+
+
+class OpenEdXAuth(TimestampedModel):
+    """Data model Open edX oauth2 access and refresh tokens"""
```

Comment:
Possibly clearer description: "Model that stores OAuth2 tokens for authenticating Open edX API calls"